resources/packs/k8s/common.go

```diff
@@ -15,6 +15,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+type K8sObjectNotFound struct{}
+
+func (e *K8sObjectNotFound) Error() string {
```

Comment:
nit: Since we call it "k8s resource" in the log messages, shouldn't the error be called `K8sResourceNotFound`?

Reply:
Good point. Thanks.